app/repositories/submission_repository.rb

```diff
@@ -77,10 +77,18 @@ def rated_not_invited
   end
 
   def limit_to_invite_at_the_moment
-    limit = Setting.get.available_spots - invited_not_expired.length
+    limit = Setting.get.available_spots - confirmed_or_still_able_to_confirm_spots
     limit >= 0 ? limit : 0
   end
 
+  def confirmed_or_still_able_to_confirm_spots
+    confirmed.length + invited_not_expired_not_confirmed.length
```

Comment:
Now we have to use `invited_not_expired_not_confirmed` instead of `invited_not_expired` here so that `confirmed` won't get counted twice, do I understand that correctly?